Controller/RegistrationController.php

```diff
@@ -162,7 +162,7 @@ public function confirmAction(Request $request, $token)
     /**
      * Tell the user his account is now confirmed.
      */
-    public function confirmedAction()
+    public function confirmedAction(Request $request)
```

Comment:
this is a BC break for anyone extending the controller already, as it changes the signature.

And it does not provide you new features, as using the RequestStack already allows you to get the Request. So I'm -1 on this PR.